addons/mail/models/mail_message.py

```diff
@@ -1113,10 +1113,13 @@ def _message_notifications_to_store(self, store: Store):
                 ),
                 Store.One(
                     "thread",
-                    Store.Attr(
-                        "modelName",
-                        lambda thread: self.env["ir.model"]._get(thread._name).display_name,
-                    ),
+                    [
+                        Store.Attr(
+                            "modelName",
+                            lambda thread: self.env["ir.model"]._get(thread._name).display_name,
+                        ),
+                        Store.Attr("display_name", rename="name"),
```

Comment:
Note: after https://github.com/odoo/odoo/pull/190243 this can just be "display_name".